docs/install.rst

```diff
@@ -151,13 +161,17 @@ in the table below.
 Dependencies For Building Library
```

Comment:
```suggestion
Dependencies For building library
```

Reply:
How about 'Dependencies for building library'

Prepositions must be in lowercase.

@LisaDelaney

Reply:
Thanks for the reviews. I've added the suggested changes, including Roopa's here, in the latest commit.